docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+services:
+  test-db:
+    image: postgres:14
+    container_name: user-integration-test
+    ports:
+      - 5433:5432
+    environment:
+      POSTGRES_USER: postgres
```

Comment:
NIT: You can specify an `.env` file as a paraemter which is really helpful in this case, may want to consider having a `env` file so that the docker compose file doesn't have to be edited in order to change out settings

Reply:
Fixed. That's very helpful!